community/kernel/pom.xml

```diff
@@ -334,11 +334,6 @@ the relevant Commercial Agreement.
       <scope>test</scope>
     </dependency>
 
-    <dependency>		
-      <groupId>org.powermock</groupId>		
-      <artifactId>powermock-module-junit4</artifactId>		
-      <scope>test</scope>		
-    </dependency>
     <dependency>
       <groupId>org.powermock</groupId>
       <artifactId>powermock-api-mockito</artifactId>
```

Comment:
This dependency was removed in https://github.com/neo4j/neo4j/pull/5171, but in a simultaneous PR https://github.com/neo4j/neo4j/pull/5159 I used PowerMock which broke the builds and forced us to add the dependency back. This PR just rewrites a test not to use PowerMock so that we can get rid of the dependency.